cloud/testserver/cloudstore/store_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package cloudstore_test
```

Comment:
I don't think we should spent too much time writing tests for tests, but it could be useful to have just one test that uses a fully pre-initialized cloudstore with data for every type and then validates that the values in all cloud.*Response types match.

Reply:
yes, I agree, just don't know yet the best of implementing this.